public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -599,12 +604,19 @@ export class GeomapPanel extends Component<Props, State> {
         if (v.id === MapCenterID.Coordinates) {
           coord = [config.lon ?? 0, config.lat ?? 0];
         } else if (v.id === MapCenterID.Fit) {
-          const extent = getLayersExtent(layers);
+          const extent = getLayersExtent(layers, config.lastOnly ?? false);
```

Comment:
We could also just pass `config.lastOnly` here and set the default param to `false` in `getLayerExtent` definition
```
export function getLayersExtent(layers: Collection<BaseLayer>, lastOnly: boolean = false): Extent {
```